src/wp-includes/media.php

```diff
@@ -1049,6 +1048,12 @@ function wp_get_attachment_image( $attachment_id, $size = 'thumbnail', $icon = f
 			'alt'      => trim( strip_tags( get_post_meta( $attachment_id, '_wp_attachment_image_alt', true ) ) ),
 			'decoding' => 'async',
 		);
+		if ( $width ) {
```

Comment:
previously width and height were always present, this might break plugins expecting this.